### PR TITLE
fix: add srkds/Nirav

### PR DESCRIPTION
**Pull request by srkds**
- add srkds.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/877"></a>

### CODE REVIEW

**Comment by kumarsonsoff3** [comment]
Also change edit you PR name to  **"add Profile type"** or **"add your_username_here"**,
Meaning to say remove **_".json"_** from your PR name.

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add srkds/Nirav`

File: public/data/srkds.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started OpenSource Contribution",
+      "date": "October 2021",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started Contributing to opensource by participating in Hacktoberfest2021",
```
Comment (Panquesito7) [inline public/data/srkds.json:19]:
```suggestion
      "description": "Started Contributing to open source by participating in Hacktoberfest 2021"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by syedsohan — APPROVED** [review]

File: public/data/srkds.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started OpenSource Contribution",
+      "date": "October 2021",
+      "icon": "github",
+      "color": "grey",
+      "description": "Started Contributing to open source by participating in Hacktoberfest 2021"
```
Comment (krishguptadev) [inline public/data/srkds.json:19]:
```suggestion
      "description": "Started contributing to open source by participating in Hacktoberfest 2021"
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by srkds** [comment]
Great 😀  🙌
Thank you for your support all reviewers 👍

**Pull request merged**